### PR TITLE
Update ir.m3u

### PR DESCRIPTION
**Pull request by data14a**

### CODE REVIEW

File: channels/ir.m3u [hunk 1]
```diff
@@ -2,13 +2,23 @@
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/Icob3E3.jpg" group-title="Entertainment",4U TV
 http://live.4ulive.ir/
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.parsatv.com/index_files/channels/afghantv.jpg" group-title="",AFN TV
-https://unirtmp.tulix.tv/tintv2/tintv2/playlist.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: This link does not require replacement.

File: channels/ir.m3u [hunk 2]
```diff
@@ -35,19 +45,25 @@ http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u8
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/ChannelIcons/gemusa-icon.png" group-title="Entertainment",GEM USA
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_usa/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/YWgJdew.png" group-title="",Glory TV
+http://glorytv.ashttp9.visionip.tv/live/glorytv-glorytv-glorytv-hsslive-25f-4x3-SD/chunklist.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: Link is broken (tested in VLC and streamtest.in)

File: channels/ir.m3u [hunk 3]
```diff
@@ -35,19 +45,25 @@ http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/ft8edTx.png" group-title="Food",HealthMedia
 https://livestream.5centscdn.com/cls092118/8f8400b79584fa2a1315aa5e7612d4f3.sdp/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/clLs5In.jpg" group-title="News",Hispan TV
+https://5a61de8ed719d.streamlock.net:1443/live/smil:live.smil/chunklist.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: Link is broken (tested in VLC and streamtest.in)

File: channels/ir.m3u [hunk 4]
```diff
@@ -35,19 +45,25 @@ http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="http://www.hodhodfarsi.tv/style-index/hodhod-logo.png" group-title="Kids",Hodhod
 http://51.158.184.234/hls/stream.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.parsatv.com/index_files/channels/icctv.jpg" group-title="Religious",ICC TV
-https://mn-nl.mncdn.com/ezcdn_icc/icc_1/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/RPwKZUD.jpg" group-title="Movies",iFILM Arabic
+https://live2.presstv.com/ifilmlive/smil:ifilmar.smil/chunklist_w1579167097_b644100_slen.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: Link is broken (tested in VLC and streamtest.in)

File: channels/ir.m3u [hunk 5]
```diff
@@ -35,19 +45,25 @@ http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/GgmXZRU.png" group-title="Movies",iFILM English
 http://mn-nl.mncdn.com/ifilm/smil:ifilmen.smil/chunklist_b1096000.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/GgmXZRU.png" group-title="Movies",iFILM English
 http://mn-nl.mncdn.com/ifilm/smil:ifilmen.smil/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/GgmXZRU.png" group-title="Movies",iFILM English
 https://bozztv.com/36bay2/gin/giniko_ifilmenglish_800kb_36bay2/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/El4Ymh6.png" group-title="Movies",iFILM Persian 1
+https://5a61de8ed719d.streamlock.net:1443/ifilmlive/smil:ifilmtv.smil/chunklist_w205801397_b944100_slen.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/El4Ymh6.png" group-title="Movies",iFILM Persian 1
```
Comment (freearhey) [inline channels/ir.m3u]: Link is broken (tested in VLC and streamtest.in)

File: channels/ir.m3u [hunk 6]
```diff
@@ -74,15 +92,17 @@ https://livetv.persianleague.com/live/tv3/chunks.m3u8
 #EXTINF:-1 tvg-id="IRIB TV4 IR" tvg-name="IRIB TV4 IR" tvg-language="Persian" tvg-logo="https://i.imgur.com/pTJEpeJ.png" group-title="General",IRIB TV4
 https://bozztv.com/36bay2/gin/giniko_irib4_900kb_36bay2/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="IRIB TV5 IR" tvg-name="IRIB TV5 IR" tvg-language="Persian" tvg-logo="https://i.imgur.com/qQBqGGf.png" group-title="General",IRIB TV5
-https://bozztv.com/36bay2/gin/giniko_irib5_900kb_36bay2/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="Sport",IRIB Varzesh
-http://cdn1.live.irib.ir:1935/channel-live/smil:varzesh.smil/chunklist_b307200_slFa.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/fFnEHc7.png" group-title="Sport",IRIB Varzesh
+https://bozztv.com/36bay2/gin/gin1iko_irib5_900kb_36bay2/tracks-v1a1/mono.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: Link is broken (tested in VLC and streamtest.in)

File: channels/ir.m3u [hunk 7]
```diff
@@ -74,15 +92,17 @@ https://livetv.persianleague.com/live/tv3/chunks.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://upload.wikimedia.org/wikipedia/en/a/a6/IRIB_Varzesh_Logo.png" group-title="Sport",IRIB Varzesh
 https://livetv.persianleague.com/live/varzesh/chunks.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="http://www.israelparstv.com/wp-content/uploads/2018/10/Israel-Pars-Logo-100x100.png" group-title="Entertainment",Israel Pars TV 
+http://66.160.192.219:8080/asaltv2/live/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/HQjLbwm.png" group-title="Religious",Kalemeh TV
 https://live.kalemehtv.tv/live/ngrp:kalemeh_all/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala Live 1
-https://stream.alkafeel.net/live/alkafeel/rAa5PGot1/manifest.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: This link does not require replacement.

File: channels/ir.m3u [hunk 8]
```diff
@@ -92,17 +112,38 @@ https://stream.alkafeel.net/live/alkafeel/rAa5PGot5/manifest.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/espyPNl.png" group-title="Religious",Mohabat TV
 http://204.11.235.251:1935/live_transcoder/ngrp:mohabat.stream_all/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/r1VblfT.png" group-title="Religious",Mohabat TV
 https://5acf9f9415a10.streamlock.net:443/live_transcoder/ngrp:mohabat.stream_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.parsatv.com/index_files/channels/omidjavedan.jpg" group-title="",Omid Javedan
+http://livestream.5centscdn.com/shaditv/23abe62a446fc05ce0a6c810f4045308.sdp/chunks.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://telegramindex.com/assets/telegram/chat/OXIRTV" group-title="Entertainment",Oxir TV
+http://212.47.226.211/hls/stream.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://live-tv-channels.org/pt-data/uploads/logo/ca-parnian-tv-9509.jpg" group-title="",Parnian TV
-https://live2.parnian.tv/hls/.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: This link works fine and replacement is not.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again and update the pull request if necessary so that we can approve it.

File: channels/ir.m3u (hunk 1 above)
Comment (data14a) [reply channels/ir.m3u]: Just an alternative link

File: channels/ir.m3u (hunk 2 above)
Comment (data14a) [reply channels/ir.m3u]: Removed

File: channels/ir.m3u (hunk 3 above)
Comment (data14a) [reply channels/ir.m3u]: Changed to a working stream

File: channels/ir.m3u (hunk 7 above)
Comment (data14a) [reply channels/ir.m3u]: Replaced with original link.

File: channels/ir.m3u (hunk 8 above)
Comment (data14a) [reply channels/ir.m3u]: This link is just a short video loop.
The link I provided is good link but does not stream 24hrs a day.

File: channels/ir.m3u (hunk 5 above)
Comment (data14a) [reply channels/ir.m3u]: Link Replaced

File: channels/ir.m3u (hunk 4 above)
Comment (data14a) [reply channels/ir.m3u]: Just tried with VLC and on Firestick and link is OK.

File: channels/ir.m3u (hunk 6 above)
Comment (data14a) [reply channels/ir.m3u]: Replaced link

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Well done!

**Pull request merged**